test/pjrt/test_ddp.py

```diff
@@ -16,7 +19,23 @@ def _init_xla_backend(init_file: str):
       init_method=f"file://{init_file}",
       rank=rank,
       world_size=world_size)
+  return rank, world_size
 
+def _assert_all_close(parameters_a, parameters_b):
```

Comment:
Will [`numpy.testing.assert_allclose`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html#numpy.testing.assert_allclose) or [`absltest`'s `assertSequenceAlmostEqual`](https://github.com/abseil/abseil-py/blob/a0ae31683e6cf3667886c500327f292c893a1740/absl/testing/absltest.py#L1000-L1021) work here instead?

Reply:
`nn.module.parameters()` is a generator. Therefore, normal array check helpers won't work. https://pytorch.org/docs/stable/_modules/torch/nn/modules/module.html#Module.parameters.